apps/snitch_core/lib/core/data/model/wishlist_item.ex

```diff
@@ -0,0 +1,72 @@
+defmodule Snitch.Data.Model.WishListItem do
+  @moduledoc """
+  API for User Wishlists
+
+  Exposes functions for wishlist feature.
```

Comment:
Remove this line, and since a user has only _one_ wishlist, L3 should be singular.